web/crux-ui/src/elements/dyo-card.tsx

```diff
@@ -4,8 +4,11 @@ import { RefAttributes } from 'react'
 export interface DyoCardProps extends RefAttributes<HTMLDivElement> {
   className?: string
   children: React.ReactNode
+  modal?: boolean
 }
 
 export const DyoCard = (props: DyoCardProps) => {
-  return <div className={clsx(props.className ?? 'p-8', 'card rounded-lg shadow-lg bg-medium')}>{props.children}</div>
+  const boxShadow = props.modal ? 'shadow-modal' : 'shadow-lg'
```

Comment:
we don't need to lift this out from clsx

Reply:
I thought it was more readable, you think I have to add back in the clsx func?